src/Durable/OrchestrationInvoker.cs

```diff
@@ -19,6 +20,16 @@ public Hashtable Invoke(OrchestrationBindingInfo orchestrationBindingInfo, IPowe
             try
             {
                 var outputBuffer = new PSDataCollection<object>();
+                var context = orchestrationBindingInfo.Context;
+
+                // context.History should never be null when initializing CurrentUtcDateTime
+                if (context.History != null)
```

Comment:
If context.History is not supposed to be null, let's remove this check and adjust the tests accordingly, as we discussed. If it ever _happens_ to be null, we should not hide this fact, let the code throw an exception.